backend/organization/models/project.py

```diff
@@ -207,10 +207,12 @@ class Project(models.Model):
 
     @property
     def cached_ranking(self) -> int:
+        """Carful with this property, it might trigger an N+1 query problem"""
         cache_key = generate_project_ranking_cache_key(project_id=self.id)
         project_rank = cache.get(cache_key)
         if not project_rank:
-            return self.rating
+            return self.ranking
+            # return self.rating
         return project_rank
```

Comment:
switched to ranking so that cache misses trigger a new rank calculation which triggers a cache.set